### ISSUE TITLE
fingers opens in different directory

### ISSUE THREAD

**Issue by ndsizeif** [Morantron/tmux-fingers#120]
Excellent project!

When tmux-fingers creates a new-window, the path can differ from the original pane. Fingers seems to be using `#{session_path}`.  If there is a repository in the tmux session path, it can be a different repository than the pane current path, which may lead to momentary confusion.

![issue](https://github.com/Morantron/tmux-fingers/assets/84802637/54538f04-c077-4ef3-8fa7-9d9f4ffe98b5)

Adding the start directory flag `-c #{pane_current_path}` keeps the directories consistent. I just threw that escape character in front of the `#`, I don't know if that's the proper way to do it in the language.

```
# tmux.cr:163
  def create_window(name, cmd, _pane_width, _pane_height)
    #output = exec("new-window -P -d -n '#{name}' -F '#{WINDOW_FORMAT}' '#{cmd}'").chomp
    output = exec("new-window -c '\#{pane_current_path}' -P -d -n '#{name}' -F '#{WINDOW_FORMAT}' '#{cmd}'").chomp
```

**Comment by Morantron** [comment]
Hey! 👋  what status bar variable are you using to display that path? I want to test the fix and make sure it works properly

**Comment by Morantron** [comment]
Fixed in 2.2.0
